packages/html-reporter/src/testCaseView.tsx

```diff
@@ -19,19 +19,26 @@ import * as React from 'react';
 import { TabbedPane } from './tabbedPane';
 import { AutoChip } from './chip';
 import './common.css';
-import { ProjectLink } from './links';
+import { Link, ProjectLink } from './links';
 import { statusIcon } from './statusIcon';
 import './testCaseView.css';
 import { TestResultView } from './testResultView';
 import { linkifyText } from './renderUtils';
 import { hashStringToInt, msToString } from './utils';
+import * as icons from './icons';
+import { useSearchParams } from './useSearchParams';
 
 export const TestCaseView: React.FC<{
   projectNames: string[],
   test: TestCase | undefined,
   anchor: 'video' | 'diff' | '',
   run: number,
-}> = ({ projectNames, test, run, anchor }) => {
+  prevTestId: string
+  nextTestId: string
+}> = ({ projectNames, test, run, anchor, prevTestId, nextTestId }) => {
+  const searchParams = useSearchParams();
+  const q = searchParams.get('q') ?? '';
```

Comment:
Making this component search params aware is unfortunate, can you pass the prev and next link hrefs instead?